### DISCUSSION TITLE
Default skin improvement suggestions

### DISCUSSION THREAD

**Discussion by jack5github** [ppy/osu#12974] [Ideas] [answered]
I have no understanding of osu!lazer's code, but I am at least partially decent at altering graphics, so here are the issues that pertain to that.

**osu!**
- [ ] Approach circles are too dim.
- [ ] Spinners are not very noticeable within gameplay due to their lack of outline and duration circle, unlike the stable osu! program which has these things.

**Taiko**
- [ ] Circle colours are too similar, not colour-blind friendly.
- [ ] Spinner has an outline but lacks a duration circle.

**osu!mania**
- [ ] Mania notes are too vertically thin.
- [ ] Mania hold notes start with a note, but end with the same note instead of with a hold drop-off. This has been a part of osu!mania since the stable program, but it would be nice to see an option that switches out hold note ends with actual ends instead of a repeated note.

There are other bugs that I found but am not willing to work on, so I guess I can't mention them here. You can find those in [this video](https://vid.me/OXcMJ) and use them in your own report if you wish.

**Comment by peppy** [comment]
That said, most of these are valid design points. Probably worth further discussion (ping @arflyte).

**Comment by KimoriWasTaken** [comment]
If I may also suggest. 
Please try to use the pink hitcircle color that you already use in Taiko (it was also in your recent youtube video @peppy https://youtu.be/5x7VnC1R0Do?t=510), instead of the red color of the current osu!standard default skin. 
I think that would make reading the new default skin a lot better since the current red is too dark to read properly.

Also may I ask in which file the osu!standard skin colours are assigned? I already searched quite a lot but the best I could find was https://github.com/ppy/osu-resources/tree/master/osu.Game.Resources/Textures/Play/osu

Thank you in advance!

**Comment by Hanamuke** [comment]
This is old, but I find the design of osu!catch bananas too similar to that of other fruits. If you're playing a hard map, a banana shower looks just like a part dense in fruits at first glance.

**Comment by peppy** [comment]
Yeah, the whole catch skin will probably get a second pass. I'm not completely happy with it either.

**Comment by SamCyanide** [comment]
I'd like to add that I think the taiko circle (where you're supposed to hit the note) in the default skin is way too thin, it is very difficult to see, at least for me. I have around 2000 plays on Taiko in the current client and it's much more difficult to play in lazer because of this.

**Comment by frenzibyte** [comment, marked as answer]
All points brought up on this thread should be resolved in the [latest gameplay skin iteration](https://www.figma.com/file/up6guVobzpc3DZlQxWzVmv/Gameplay-Layout), and implementation will be tracked under https://github.com/ppy/osu/issues/18887.
